### DISCUSSION TITLE
tell your Fav 40k faction for Galaxy brain

### DISCUSSION THREAD

**Discussion by slyUDgarloki** [Schweinepriester/github-profile-achievements#605] [Q&A] [answered]
Mine is Custodians and Ultramarine and Death Guard

**Comment by pranjal779** [comment, marked as answer]
Favorite in order:
Death Korps of Krieg
Alpha Legion
Tau
Ultramarines
Death Guard
Imperial Fists
Iron Hands
Space Wolves

**Reply by slyUDgarloki** [reply]
nice
